app/models/suggestion.py

```diff
@@ -1,22 +1,27 @@
 from datetime import datetime
 import pytz
+
 from .. import db
+from ..models.resource import Resource, ResourceBase
+
 
-class Suggestion(db.Model):
+class ResourceSuggestion(ResourceBase):
     """
     Association between a resource and potential suggestions for it
```

Comment:
chance comment to reflect " a suggestion to either create a new resources OR edit an existing resource"